wcfsetup/install/files/js/WoltLabSuite/Core/Controller/Clipboard.js

```diff
@@ -1,201 +1,158 @@
 /**
  * Clipboard API Handler.
  *
- * @author	Alexander Ebert
- * @copyright	2001-2019 WoltLab GmbH
- * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
- * @module	WoltLabSuite/Core/Controller/Clipboard
+ * @author  Alexander Ebert
+ * @copyright  2001-2019 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
```

Comment:
Unfortunately this diff is too large to review :disappointed:

Reply:
It was a bit of a nightmare, but I spent quite some time testing it, therefore I'm somewhat confident it should be fine.